src/cursorless.talon

```diff
@@ -1,64 +1,64 @@
 app: vscode
 -
 {self.simple_cursorless_action} <user.cursorless_target>:
-    user.cursorless_single_target_command(simple_cursorless_action, cursorless_target)
+	user.cursorless_single_target_command(simple_cursorless_action, cursorless_target)
 
 def show <user.cursorless_target>:
-    user.cursorless_single_target_command("setSelection", cursorless_target)
-    user.vscode("editor.action.revealDefinition")
+	user.cursorless_single_target_command("setSelection", cursorless_target)
+	user.vscode("editor.action.revealDefinition")
 
 ref show <user.cursorless_target>:
-    user.cursorless_single_target_command("setSelection", cursorless_target)
-    user.vscode("references-view.find")
+	user.cursorless_single_target_command("setSelection", cursorless_target)
```

Comment:
we should probably align on whitespace format for the talon files.  I'm generally not in love with tabs but willing to be convinced otherwise 😊

Reply:
I can change to whitespaces right away. I've already been thinking about it but was not got around to it yet.

Reply:
So done! The ext is updated and the next commit will have spaces instead.

Reply:
These still seem to be spaces.  Shall we revert them to original spaces before merging?

Reply:
Fixed now.